pkg/maintainer/spv/spv.go

```diff
@@ -59,8 +63,271 @@ func (sm *spvMaintainer) startControlLoop(ctx context.Context) {
 func (sm *spvMaintainer) maintainSpv(ctx context.Context) error {
 	logger.Infof("Maintaining SPV proof...")
```

Comment:
I would drop this line given we log earlier:

https://github.com/keep-network/keep-core/blob/8030414dee6ea771e69b21278163e8aee1cb6319/pkg/maintainer/spv/spv.go#L40

Reply:
Done in 32150aba63c2cea884d25fa231532958169bdcb1.